lib/win32/taskscheduler.rb

```diff
@@ -193,7 +193,30 @@ def enum
     # Returns whether or not the specified task exists.
     #
     def exists?(task)
-      enum.include?(task)
+      path = nil
+      task_name = nil
+
+      if task.include?("\\")
+        *path, task_name = task.split("\\")
+      end
+
+      folder = path.nil? ? "\\" : path.join("\\")
+
+      begin
+        root = @service.GetFolder(folder)
+      rescue WIN32OLERuntimeError => err
+        return false
+      end
+
+      task = nil
```

Comment:
if you can avoid using `task` for two different purposes in this method you can drop this nil.